### PR TITLE
mx: set default graal.options.file property to graal.options.

### PR DESCRIPTION
**Pull request by zapster**
/cc @dougxc

### CODE REVIEW

**Comment by zapster** [comment]
Is this change worth a ChangeLog entry?

File: mx.graal-core/mx_graal_8.py [hunk 1]
```diff
@@ -400,20 +400,23 @@ def _getDeployedJars():
...
+            if '=' in arg:
+                mx.abort('Mixing - and = in -G: option specification: ' + arg)
+            arg = '-Dgraal.' + arg[len('-G:+'):] + '=false'
+        elif arg.startswith('-G:'):
+            if '=' not in arg:
+                mx.abort('Missing "=" in non-boolean -G: option specification: ' + arg)
+            arg = '-Dgraal.' + arg[len('-G:'):]
+        return arg
+    # add default graal.options.file and translate -G: options
+    return ['-Dgraal.options.file=graal.options'] + map(_translateGOption, args)
```
Comment (tkrodriguez) [inline mx.graal-core/mx_graal_8.py]: Shouldn't this be an absolute path into the suite?  Some of the mx commands change their directory before running things so it won't be found in this case.

Comment (dougxc) [inline mx.graal-core/mx_graal_8.py]: In addition, we should only add the system property to the command line if the file actually exists.

**Comment by dougxc** [comment]
No need for a changelog entry as this is fixing a regression in expected behavior.

File: mx.graal-core/mx_graal_8.py (hunk 1 above)
Comment (zapster) [inline mx.graal-core/mx_graal_8.py]: @tkrodriguez  right, suite path is probably better.
@dougxc ok, can do.

**Pull request merged**